### ISSUE TITLE
Help wanted: Pluralization for russian

### ISSUE THREAD

**Issue by ElForastero** [i18next/i18next-icu#9]
Can't understand what I'm doing wrong, but pluralization doesn't work as expected.

It works, but only for `one` and `other` cases, as in English, despite the fact that Russian language has 4 plural forms.

ICU format is correct.

`{count, number} {count, plural, one {one} few {few} many {many} other {other}}`

Iterating through numbers from 1 to 200 I'm getting the next results:
```
1 one
2 other
3 other
4 other
5 other
6 other
7 other
8 other
9 other
10 other
11 other
12 other
13 other
14 other
15 other
16 other
17 other
18 other
19 other
20 other
21 other
22 other
...
```
Definition of plural forms:
http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html#ru

Sandbox with example: https://codesandbox.io/s/lypy5z7pkz

**Comment by jamuhl** [comment]
Hm...guess passing localeData as an array is a lie...does only work if passing 

```js
  .init({
    i18nFormat: {
      localeData: ru
    },
```

https://github.com/i18next/i18next-icu/blob/master/src/index.js#L29

will fix this on monday...

**Comment by jamuhl** [comment]
fixed in i18next-icu@1.1.2

**Comment by jamuhl** [comment]
If you like this module don’t forget to star this repo. Make a tweet, share the word or have a look at our https://locize.com to support the devs of this project -> there are many ways to help this project :pray:
